setup.cfg

```diff
@@ -15,22 +15,23 @@ classifiers =
   Environment :: Console
   Intended Audience :: Science/Research
   Operating System :: OS Independent
-  Programming Language :: Python :: 3.6
-  Programming Language :: Python :: 3.7
   Programming Language :: Python :: 3.8
   Programming Language :: Python :: 3.9
+  Programming Language :: Python :: 3.10
+  Programming Language :: Python :: 3.11
+  Programming Language :: Python :: 3.12
   Programming Language :: Python :: 3 :: Only
   Topic :: Utilities
 license_file = LICENSE
 long_description = file: README.md
 long_description_content_type = text/markdown
 
 [options]
-python_requires = >= 3.6
+python_requires = >= 3.8
 include_package_data = True
 packages = find:
 install_requires =
-  numpy
+  numpy >= 1.21
```

Comment:
This was the oldest version of numpy I could get  the tests to pass with, primarily because of `numpy.typing` I think.